packages/@aws-cdk/cfnspec/build-tools/patch-stack.ts

```diff
@@ -0,0 +1,232 @@
+
+/**
+ * Apply a JSON patch stack into the given target file
```

Comment:
`~` I would have gone with `patch chain` or `patch sequence` or something like that. `Stack` is such an overloaded term in the CDK that it's hard to read `patch-stack` and not be thinking about CloudFormation stacks (or stack sets, or CDK stacks, or ...).